### DISCUSSION TITLE
What should I do if I want to set a production well as phaseVolRate type?

### DISCUSSION THREAD

**Discussion by liushilongpku** [GEOS-DEV/GEOS#3380] [Q&A]
Hello everyone. I have meet some troubles when I'm tring to set well control type to phaseVolRate. I have case which need to produce air at a fix volume rate. When the target BHP pressure is not low enough, the well couldn't work at the right rate, the well preesure will be not low than the value which I have set. But if I set the  target BHP pressure is low enough, it will be not easy to converge at the beging time when appling the production well. It seems like the target BHP pressure was been used as the initial condition of the production well. So what should i do if i want to set a production well as phaseVolRate type?

**Comment by tjb-ltk** [comment]
Typically a well model would pick the most limiting constraint... for a producer it would be the constraint with the highest BHP, injector lowest.    Can you post the input file , it will be easier to see what sort of convergence issues you are having...
